### PR TITLE
added trace

### PR DESCRIPTION
**Pull request by Kazeno21**
Issue:

### CODE REVIEW

**Comment by Kazeno21** [comment]
@shahednasser 
issue #420

File: src/components/animated/_trace.less [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.trace-btn{
+    border:5px solid black;
+    padding: 1.5rem;
```
Comment (shahednasser) [inline src/components/animated/_trace.less:3]: Please remove the width and the padding as the sizing of the buttons shouldn't change

File: assets/js/buttons-examples.js [hunk 2]
```diff
@@ -26,6 +26,11 @@ $(document).ready(function () {
       label: "Animated Buttons",
       classes: "",
       children: [
+        {
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:29]: Place this button in the correct alphabetical order under Animated

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for your work the button is looking much butter now. I see though that it does not work as it should on the rounded buttons, can you fix that please? Also I'm not a big fan of the color change so either change the colors that are being changed or use just one color.

**Comment by Kazeno21** [comment]
@shahednasser, i have tried my best to resolve the issues pointed out by you. I have changed the border color to black and also placed the button in alphabetical order. I have tried a lot to resolve the rounded buttons issue but couldn't do it. And wrt the width, if i'm not keeping it fixed, the button for some reason takes the width of the entire screen despite of using wrap. Can you please guide me?

**Review by shahednasser — CHANGES_REQUESTED** [review]
I'm not sure what problem you are having with the width, as I tried removing it and it looked fine.

As for the border in the rounded buttons I'm honestly not sure if I can help considering that you are using an svg border image. I'm not sure if you made it yourself or you copied it from somewhere (both is fine) but you should be able to figure it out more than I can.

**Comment by Kazeno21** [comment]
> I'm not sure what problem you are having with the width, as I tried removing it and it looked fine.
> 
> As for the border in the rounded buttons I'm honestly not sure if I can help considering that you are using an svg border image. I'm not sure if you made it yourself or you copied it from somewhere (both is fine) but you should be able to figure it out more than I can.

@shahednasser, yes this is my first time working with css animations so i took a little help from the internet. i tried the animation code without using svg but i wasn't sure how to integrate that hence i used svg. please guide me further.

**Comment by Kazeno21** [comment]
@shahednasser can you please resolve the conflicts and merge this. then you can create a new issue for the changes required and assign it to someone else. Please. This will be really helpful and give me an opportunity to learn from someone better.

**Comment by shahednasser** [comment]
@Kazeno21 I can't really merge it as it isn't fully working. I know you need to learn but this would also be a great opportunity to learn how to fix this. You can take your time with it you don't have to rush it.

**Comment by Kazeno21** [comment]
@shahednasser , thanks a lot for being so supportive because i was under the impression that I am a hinderance to your work and causing delays.

**Comment by shahednasser** [comment]
@Kazeno21 not at all! All issues labeled with Priority Low can wait! Take your time and figure this out.

Thank you for your time.

**Comment by shahednasser** [comment]
Hi @Kazeno21 I will be closing this PR as it is very outdated. If you still want to submit your work, please submit a new PR. Thank you.

**Pull request closed without merging**